Fix describeEnvironements issue

antoinegoutagny opened the pull request:
For some reason, including the version label in the describeEnvironments request returns no result. This change fixes that, and checks the results for the correct version label.

simoneb commented:
@antoinegoutagny thanks for the PR. Unfortunately I have no way to test this (not using aws eb any longer) so I won't merge this PR and won't release a new version of the plugin because I cannot test it.

If you are an AWS eb user and would like to take over the development of the plugin, you are more than welcome.

antoinegoutagny commented:
Well this package is an important part of our deployment workflow, so yes I'm interested in transfering ownership. What's the procedure to do that?

simoneb commented:
github has a mechanism to transfer ownership of repositories between users, I can do it and transfer it to you ifyou are ok with it

antoinegoutagny commented:
Yes I'm ok with that. Please transfer it to Swaven (my compagny) rather than my personal account.
Is there an impact if we already have a fork of the repo?

simoneb commented:
You can handle the fork yourself I guess (it's just another repo). I'll try the transfer now, the owner of the account should receive a notification and needs to accept the transfer

simoneb commented:
Ah, unfortunately because you have already a fork the transfer cannot happen. I'm in doubt whether we should either:

- make yours the *official* fork (how to inform people especially)
- delete your fork, transfer mine to you, re-integrate your changes

Specifically, I'm not sure if going for the second option (which takes a little more work) would help with informing people that your fork is now the official one. Ideas?

antoinegoutagny commented:
I don't think the 2nd option will help with informing peopple of the
change, since anyway its url will change. The only downside I see with 1st
option is that the 2 other pull requests on your repo will be lost. But I
don't think that's a huge issue since they've been sitting there for quite
some time, I'm not sure they're even still valid.
Option 1 would only be changing the repo source on npm I guess, right?

2015-10-09 15:33 GMT+02:00 Simone Busoli <notifications@github.com>:

> Ah, unfortunately because you have already a fork the transfer cannot
> happen. I'm in doubt whether we should either:
>
>    - make yours the *official* fork (how to inform people especially)
>    - delete your fork, transfer mine to you, re-integrate your changes
>
> Specifically, I'm not sure if going for the second option (which takes a
> little more work) would help with informing people that your fork is now
> the official one. Ideas?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/simoneb/grunt-awsebtdeploy/pull/13#issuecomment-146872692>
> .
>

simoneb commented:
I also think the 2 PRs won't be an issue. As for the package on npmjs, yes the repo needs to change but also you need permissions to publish new releases. On npmjs I don't see a way to transfer ownership of a package to someone else but only to add collaborators, which I just did with swaven.

Once you're set I'll just add a notice to the readme file to point to your fork.

Simone

antoinegoutagny commented:
Ok, I was able to publish a new version to npm (that's a first for me),
including version bump with our changes. Is there anything else I need to
do?

2015-10-09 15:50 GMT+02:00 Simone Busoli <notifications@github.com>:

> I also think the 2 PRs won't be an issue. As for the package on npmjs, yes
> the repo needs to change but also you need permissions to publish new
> releases. On npmjs I don't see a way to transfer ownership of a package to
> someone else but only to add collaborators, which I just did with swaven.
>
> Once you're set I'll just add a notice to the readme file to point to your
> fork.
>
> Simone
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/simoneb/grunt-awsebtdeploy/pull/13#issuecomment-146876414>
> .
>

simoneb commented:
nope, we're done. thanks!

antoinegoutagny commented:
BTW, do you wish to be kept as the package author, or would you rather be
removed? I've added myself as contributor, but I can change that if you
prefer to not be associated with it anymore.

2015-10-09 16:18 GMT+02:00 Simone Busoli <notifications@github.com>:

> Closed #13 <https://github.com/simoneb/grunt-awsebtdeploy/pull/13>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/simoneb/grunt-awsebtdeploy/pull/13#event-431456962>.
>

simoneb commented:
I should still appear as the author, being the author, just not the
maintainer, so all mentions of my name should remain in place.

On Fri, Oct 9, 2015 at 4:33 PM, Antoine Goutagny <notifications@github.com>
wrote:

> BTW, do you wish to be kept as the package author, or would you rather be
> removed? I've added myself as contributor, but I can change that if you
> prefer to not be associated with it anymore.
>
> 2015-10-09 16:18 GMT+02:00 Simone Busoli <notifications@github.com>:
>
> > Closed #13 <https://github.com/simoneb/grunt-awsebtdeploy/pull/13>.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/simoneb/grunt-awsebtdeploy/pull/13#event-431456962>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/simoneb/grunt-awsebtdeploy/pull/13#issuecomment-146889145>
> .
>

antoinegoutagny commented:
Sure, no problem. Just asking because I'm not really familiar with open
source conventions :).

2015-10-09 16:35 GMT+02:00 Simone Busoli <notifications@github.com>:

> I should still appear as the author, being the author, just not the
> maintainer, so all mentions of my name should remain in place.
>
> On Fri, Oct 9, 2015 at 4:33 PM, Antoine Goutagny <notifications@github.com
> >
> wrote:
>
> > BTW, do you wish to be kept as the package author, or would you rather be
> > removed? I've added myself as contributor, but I can change that if you
> > prefer to not be associated with it anymore.
> >
> > 2015-10-09 16:18 GMT+02:00 Simone Busoli <notifications@github.com>:
> >
> > > Closed #13 <https://github.com/simoneb/grunt-awsebtdeploy/pull/13>.
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/simoneb/grunt-awsebtdeploy/pull/13#event-431456962
> >.
> >
> > >
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/simoneb/grunt-awsebtdeploy/pull/13#issuecomment-146889145
> >
>
> > .
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/simoneb/grunt-awsebtdeploy/pull/13#issuecomment-146889993>
> .
>

simoneb commented:
not a problem. thanks for taking over the project!

On Fri, Oct 9, 2015 at 4:41 PM, Antoine Goutagny <notifications@github.com>
wrote:

> Sure, no problem. Just asking because I'm not really familiar with open
> source conventions :).
>
> 2015-10-09 16:35 GMT+02:00 Simone Busoli <notifications@github.com>:
>
> > I should still appear as the author, being the author, just not the
> > maintainer, so all mentions of my name should remain in place.
> >
> > On Fri, Oct 9, 2015 at 4:33 PM, Antoine Goutagny <
> notifications@github.com
> > >
> > wrote:
> >
> > > BTW, do you wish to be kept as the package author, or would you rather
> be
> > > removed? I've added myself as contributor, but I can change that if you
> > > prefer to not be associated with it anymore.
> > >
> > > 2015-10-09 16:18 GMT+02:00 Simone Busoli <notifications@github.com>:
> > >
> > > > Closed #13 <https://github.com/simoneb/grunt-awsebtdeploy/pull/13>.
> > > >
> > > > —
> > > > Reply to this email directly or view it on GitHub
> > > > <
> https://github.com/simoneb/grunt-awsebtdeploy/pull/13#event-431456962
> > >.
> > >
> > > >
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> >
> https://github.com/simoneb/grunt-awsebtdeploy/pull/13#issuecomment-146889145
> > >
> >
> > > .
> > >
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/simoneb/grunt-awsebtdeploy/pull/13#issuecomment-146889993
> >
>
> > .
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/simoneb/grunt-awsebtdeploy/pull/13#issuecomment-146891647>
> .
>
